### ISSUE TITLE
Exception on /pa settings with spigot 1.8

### ISSUE THREAD

**Issue by Cheos137** [A5H73Y/Parkour#273]
The ``/pa settings`` command cannot execute due to a ClassNotFoundException (org.bukkit.util.Consumer) on my spigot 1.8 server.

Plugin version: 6.3
spigot version: git-Spigot-550ebace-7019900e (MC: 1.8)

**Comment by Cheos137** [comment]
Full stacktrace:
```
21.02 10:23:01 [Server] ERROR null
21.02 10:23:01 [Server] INFO org.bukkit.command.CommandException: Unhandled exception executing command 'pa' in plugin Parkour v6.3
21.02 10:23:01 [Server] INFO at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot_1.8.jar:git-Spigot-550ebace-7019900e]
21.02 10:23:01 [Server] INFO at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[spigot_1.8.jar:git-Spigot-550ebace-7019900e]
21.02 10:23:01 [Server] INFO at org.bukkit.craftbukkit.v1_8_R1.CraftServer.dispatchCommand(CraftServer.java:646) ~[spigot_1.8.jar:git-Spigot-550ebace-7019900e]
21.02 10:23:01 [Server] INFO at net.minecraft.server.v1_8_R1.PlayerConnection.handleCommand(PlayerConnection.java:1115) [spigot_1.8.jar:git-Spigot-550ebace-7019900e]
21.02 10:23:01 [Server] INFO at net.minecraft.server.v1_8_R1.PlayerConnection.a(PlayerConnection.java:950) [spigot_1.8.jar:git-Spigot-550ebace-7019900e]
21.02 10:23:01 [Server] INFO at net.minecraft.server.v1_8_R1.PacketPlayInChat.a(PacketPlayInChat.java:26) [spigot_1.8.jar:git-Spigot-550ebace-7019900e]
21.02 10:23:01 [Server] INFO at net.minecraft.server.v1_8_R1.PacketPlayInChat.a(PacketPlayInChat.java:53) [spigot_1.8.jar:git-Spigot-550ebace-7019900e]
21.02 10:23:01 [Server] INFO at net.minecraft.server.v1_8_R1.PacketHandleTask.run(SourceFile:13) [spigot_1.8.jar:git-Spigot-550ebace-7019900e]
21.02 10:23:01 [Server] INFO at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_211]
21.02 10:23:01 [Server] INFO at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_211]
21.02 10:23:01 [Server] INFO at net.minecraft.server.v1_8_R1.MinecraftServer.z(MinecraftServer.java:696) [spigot_1.8.jar:git-Spigot-550ebace-7019900e]
21.02 10:23:01 [Server] INFO at net.minecraft.server.v1_8_R1.DedicatedServer.z(DedicatedServer.java:316) [spigot_1.8.jar:git-Spigot-550ebace-7019900e]
21.02 10:23:01 [Server] INFO at net.minecraft.server.v1_8_R1.MinecraftServer.y(MinecraftServer.java:634) [spigot_1.8.jar:git-Spigot-550ebace-7019900e]
21.02 10:23:01 [Server] INFO at net.minecraft.server.v1_8_R1.MinecraftServer.run(MinecraftServer.java:537) [spigot_1.8.jar:git-Spigot-550ebace-7019900e]
21.02 10:23:01 [Server] INFO at java.lang.Thread.run(Thread.java:748) [?:1.8.0_211]
21.02 10:23:01 [Server] INFO Caused by: java.lang.BootstrapMethodError: java.lang.NoClassDefFoundError: org/bukkit/util/Consumer
21.02 10:23:01 [Server] INFO at io.github.a5h73y.parkour.gui.impl.CourseSettingsGui.addContent(CourseSettingsGui.java:61) ~[?:?]
21.02 10:23:01 [Server] INFO at io.github.a5h73y.parkour.gui.ParkourGuiManager.showMenu(ParkourGuiManager.java:43) ~[?:?]
21.02 10:23:01 [Server] INFO at io.github.a5h73y.parkour.type.course.CourseManager.displaySettingsGui(CourseManager.java:996) ~[?:?]
21.02 10:23:01 [Server] INFO at io.github.a5h73y.parkour.commands.ParkourCommands.onCommand(ParkourCommands.java:540) ~[?:?]
21.02 10:23:01 [Server] INFO at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot_1.8.jar:git-Spigot-550ebace-7019900e]
21.02 10:23:01 [Server] INFO ... 14 more
21.02 10:23:01 [Server] INFO Caused by: java.lang.NoClassDefFoundError: org/bukkit/util/Consumer
21.02 10:23:01 [Server] INFO at io.github.a5h73y.parkour.gui.impl.CourseSettingsGui.addContent(CourseSettingsGui.java:61) ~[?:?]
21.02 10:23:01 [Server] INFO at io.github.a5h73y.parkour.gui.ParkourGuiManager.showMenu(ParkourGuiManager.java:43) ~[?:?]
21.02 10:23:01 [Server] INFO at io.github.a5h73y.parkour.type.course.CourseManager.displaySettingsGui(CourseManager.java:996) ~[?:?]
21.02 10:23:01 [Server] INFO at io.github.a5h73y.parkour.commands.ParkourCommands.onCommand(ParkourCommands.java:540) ~[?:?]
21.02 10:23:01 [Server] INFO at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot_1.8.jar:git-Spigot-550ebace-7019900e]
21.02 10:23:01 [Server] INFO ... 14 more
21.02 10:23:01 [Server] INFO Caused by: java.lang.ClassNotFoundException: org.bukkit.util.Consumer
21.02 10:23:01 [Server] INFO at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:91) ~[spigot_1.8.jar:git-Spigot-550ebace-7019900e]
21.02 10:23:01 [Server] INFO at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:86) ~[spigot_1.8.jar:git-Spigot-550ebace-7019900e]
21.02 10:23:01 [Server] INFO at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_211]
21.02 10:23:01 [Server] INFO at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_211]
21.02 10:23:01 [Server] INFO at io.github.a5h73y.parkour.gui.impl.CourseSettingsGui.addContent(CourseSettingsGui.java:61) ~[?:?]
21.02 10:23:01 [Server] INFO at io.github.a5h73y.parkour.gui.ParkourGuiManager.showMenu(ParkourGuiManager.java:43) ~[?:?]
21.02 10:23:01 [Server] INFO at io.github.a5h73y.parkour.type.course.CourseManager.displaySettingsGui(CourseManager.java:996) ~[?:?]
21.02 10:23:01 [Server] INFO at io.github.a5h73y.parkour.commands.ParkourCommands.onCommand(ParkourCommands.java:540) ~[?:?]
21.02 10:23:01 [Server] INFO at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot_1.8.jar:git-Spigot-550ebace-7019900e]
21.02 10:23:01 [Server] INFO ... 14 more```

**Comment by A5H73Y** [comment]
Fixed in latest commit, will be included as part of the Parkour v6.4 release.
